dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/constants/Constants.java

```diff
@@ -735,4 +735,14 @@ private Constants() {
      * K8S sensitive param
      */
     public static final String K8S_CONFIG_REGEX = "(?<=((?i)configYaml(\" : \"))).*?(?=(\",\\n))";
+
+
+    /**
+     * netty config enable
+     */
+    public static final String NETTY_SSL_ENABLE = "netty.ssl.enable";
+
+    public static final String NETTY_SSL_KEY_PATH = "netty.ssl.key.path";
+
+    public static final String NETTY_SSL_CERT_PATH = "netty.ssl.cert.path";
```

Comment:
Pleas move this kind of config into `application.yaml`, we shouldn't  add new config into common.properties.
```
rpc:
   ssl:
       enable: true
       cert-file: 
       key-file: 
```

Reply:
i add config into common.properties,Because if the master service uses SSL authentication, the worker must also be configured。 so i think only config into common better than config both into master and worker